fix/w-13119685 – Added tip on using secret synchronization with any secret (#1025)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/custom-ingress-configuration.adoc

Applying this label to your secret enables your ingress controller to immediately apply the updated TLS material into application namespaces. If you don't add this label, you must manually update the secret in each namespace.

[NOTE]

> You can use the syncronization mechanism for any secret that you would like replicated from Runtime Fabric agent namespace into the application namespace upon deployment.

Reviewer: ```suggestion
You can use the synchronization mechanism to replicate any secret from the Runtime Fabric agent namespace into the application namespace upon deployment.
```
